mkt/ratings/tests/test_views.py

```diff
@@ -281,6 +282,42 @@ def test_already_rated_version(self):
         assert data['user']['can_rate']
         assert not data['user']['has_rated']
 
+    def test_no_lang_filter(self):
+        Review.objects.create(addon=self.app, user=self.user, body='yes')
+        other_user = UserProfile.objects.exclude(pk=self.user.pk)[0]
+        other_user.update(lang='pt-BR')
+        Review.objects.create(addon=self.app, user=other_user, body='yes')
+        res, data = self._get_url(self.list_url, app=self.app.app_slug,
+                                  client=self.anon, lang='pt-BR')
+        eq_(res.status_code, 200)
+        eq_(len(data['objects']), 2)
+        eq_(data['info']['total_reviews'], 2)
+
+    def test_lang_filter(self):
+        Review.objects.create(addon=self.app, user=self.user, body='yes')
+        other_user = UserProfile.objects.exclude(pk=self.user.pk)[0]
+        other_user.update(lang='pt-BR')
+        Review.objects.create(addon=self.app, user=other_user, body='yes')
+        res, data = self._get_url(self.list_url, app=self.app.app_slug,
+                                  client=self.anon, lang='pt-BR',
+                                  match_lang='1')
+        eq_(res.status_code, 200)
+        eq_(len(data['objects']), 1)
+        eq_(data['info']['total_reviews'], 2)
+        eq_(data['objects'][0]['lang'], 'pt-BR')
+
+    def test_lang_filter_inverse(self):
+        Review.objects.create(addon=self.app, user=self.user, body='yes')
+        other_user = UserProfile.objects.exclude(pk=self.user.pk)[0]
+        other_user.update(lang='pt-BR')
+        Review.objects.create(addon=self.app, user=other_user, body='yes')
+        res, data = self._get_url(self.list_url, app=self.app.app_slug,
+                                  client=self.anon, lang='pt-BR',
+                                  match_lang='0')
+        eq_(res.status_code, 200)
+        eq_(len(data['objects']), 2)
+        eq_(data['info']['total_reviews'], 2)
```

Comment:
Oh wait, that's what the above test is... so why doesn't it return 1 review only since it should exclude the `pt-BR` one?